### ISSUE TITLE
Long head of biceps tendon dislocation

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11006]
**Preferred term label:**
Long head of biceps tendon dislocation

**Synonyms**

**Definition (free text, please give PubMed ID)**
MRI shoulder showing dislocation or subluxation of the long head of the biceps tendon out of the bicipital groove,
OR
Dynamic ultrasound of the shoulder showing subluxation or dislocation of the long head of the biceps tendon out of the bicipital groove

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Biceps subluxation
Rotator cuff tear

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001465
- label: Long head of biceps tendon dislocation
- parents: [Abnormal tendon morphology (HP:0100261)](https://hpo.jax.org/app/browse/term/HP:0100261)
- pmid: PMID:30475566|PMID:17126231
- def: Dislocation or subluxation of the long head of the biceps tendon out of the bicipital groove (as documented by magnetic resonance imaging of the shoulder) or dynamic ultrasound of the shoulder showing subluxation or dislocation of the long head of the biceps tendon out of the bicipital groove.
- comment: Isolated dislocation of the long head of the biceps is a rare entity. In most cases, dislocation of the biceps tendon out of the bicipital groove is usually associated with tears of the subscapularis tendon or massive rotator cuff tears (or both).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11006
- synonyms: LHBT dislocation(NONE)
